# Fix some bashims in the testsuite. The fix is a bit ugly, so I think it has
# to be rewritten before sending it upstream.

diff -Nurd glibc-2.3.6.orig/linuxthreads/tst-tls2.sh glibc-2.3.6/linuxthreads/tst-tls2.sh
--- glibc-2.3.6.orig/linuxthreads/tst-tls2.sh	2003-09-02 02:29:31.000000000 +0200
+++ glibc-2.3.6/linuxthreads/tst-tls2.sh	2006-08-20 16:28:09.430784935 +0200
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 common_objpfx=$1; shift
 elf_objpfx=$1; shift
diff -Nurd glibc-2.3.6.orig/nptl/tst-tls6.sh glibc-2.3.6/nptl/tst-tls6.sh
--- glibc-2.3.6.orig/nptl/tst-tls6.sh	2003-09-03 00:02:59.000000000 +0200
+++ glibc-2.3.6/nptl/tst-tls6.sh	2006-08-20 16:28:31.045608966 +0200
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 common_objpfx=$1; shift
 elf_objpfx=$1; shift
